core/EventDispatcher.php

```diff
@@ -57,9 +64,14 @@ class EventDispatcher extends Singleton
     /**
      * Constructor.
      */
-    public function __construct($pluginManager = null)
+    public function __construct(Plugin\Manager $pluginManager, array $globalObservers = array())
```

Comment:
May I suggest a different name than `globalObservers` here: in this context (in the EventDispatcher) we shouldn't assume that it's the "global observers list", instead it's just a list of observers to register straight in the constructor. I would name it `$observers` or something more generic like that. I'm nitpicking about it only because it is used in the DI configuration